pkg/operator/sync.go

```diff
@@ -74,13 +74,18 @@ func (optr *Operator) syncClusterAPIController(config *OperatorConfig) error {
 }
 
 func (optr *Operator) syncBaremetalControllers(config *OperatorConfig) error {
-	// First create a Secret needed for the Metal3 deployment
+	// Try to get baremetal provisioning config from a CR
+	baremetalProvisioningConfig, err := getBaremetalProvisioningConfig(optr.dynamicClient, baremetalProvisioningCR)
+	if err != nil {
+		glog.Infof("Unable to read Baremetal Provisioning config from CR %s.", baremetalProvisioningCR)
```

Comment:
I find it weird not returning err here, at minimum this should log.error, and make clear in the message the function will continue nevertheless

Reply:
In this release, we cannot return error and stop deploying Metal3 because during discussions around upgrade strategy, we had decided to have one release where the config could be obtained from either sources, the ConfigMap or the newly created Config resource.
In the current implementation, we are prioritizing reading config from the new Config Resource. When that fails, (could be for a simple reason that the user did not want to provide a CR and instead still wanted to use a configmap), then we should be able to proceed with the deployment. 
The other part of the comment talks about logging an error. Again, I worry that this might be interpreted as something went wrong. If you think seeing an error msg in the logs will not be interpreted as I think it will, I am fine with making this Info statement into an Error msg.